Raspi/Test/Pipfile

```diff
@@ -1,13 +1,16 @@
 [[source]]
-url = "https://pypi.python.org/simple"
-verify_ssl = true
 name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
 
 [packages]
 paho-mqtt = "*"
-"urllib3" = "*"
-
-[dev-packages]
+scipy = "*"
+pandas = "*"
+argparse = "*"
+urllib3 = "*"
 
 [requires]
-python_version = "3.6"
+python_version = "3.8"
```

Comment:
Is there a reason why this was upgraded to 3.8? We might need to look into setting up `pyenv` on everyones machines.